wasm/src/debug_service.rs

```diff
@@ -163,8 +169,15 @@ pub struct BreakpointSpan {
 
 #[wasm_bindgen(typescript_custom_section)]
 const IStackFrameList: &'static str = r#"
+export interface IStackFrame {
```

Comment:
If you're going to export these as explicit interfaces in these custom sections, do the actual structs (StackFrame and BreakpointSpan) still need to be exposed via wasm_bindgen and pub?

Reply:
Good call, removing them.